src/proto/response.rs

```diff
@@ -1,10 +1,12 @@
+use super::error::ZkError;
+use super::request::{MultiHeader, OpCode};
 use byteorder::{BigEndian, ReadBytesExt};
 use failure;
 use std::io::{self, Read};
 use {Acl, KeeperState, Permission, Stat, WatchedEvent, WatchedEventType};
 
 #[derive(Debug)]
-pub(crate) enum Response {
+pub enum Response {
```

Comment:
If this is indeed going to become `pub`, we need to also make all the fields pub, and add documentation to all the variants. Why do you think it has to be `pub`?

Reply:
Oops, good catch. It needed to be `pub` in an earlier iteration of this patch where I exposed `Response` directly. I later decided that creating a special `MultiResponse` type for external consumption was cleaner. Fixed!

Reply:
Oops, good catch. It needed to be pub in an earlier iteration of this patch where I exposed Response directly. I later decided that creating a special MultiResponse type for external consumption was cleaner. Fixed!